admin/commands/inventory/add_agent_qan_postgres_pgstatements_agent.go

```diff
@@ -52,6 +52,7 @@ type AddAgentQANPostgreSQLPgStatementsAgentCommand struct {
 	Password            string            `help:"PostgreSQL password for QAN agent"`
 	CustomLabels        map[string]string `mapsep:"," help:"Custom user-assigned labels"`
 	SkipConnectionCheck bool              `help:"Skip connection check"`
+	QueryLength         int32             `placeholder:"NUMBER" help:"Limit query length in QAN (default: server-defined; -1: no limit)"`
```

Comment:
I suggest to name this field `MaxQueryLength` to make it clear that it defines limitation.

Reply:
Done for this PR and PR for MySQL.